shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/context/SQLUnit.java

```diff
@@ -36,4 +40,15 @@
     private final String sql;
     
     private final List<Object> parameters;
+
+    private final Set<String> actualTables;
+    
+    private final Map<String, List<String>> primaryKeyColumns;
```

Comment:
What is the useful of the structure of `Map<String, List<String>>`?
Can we use a object to define this structure?